doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -2698,17 +2699,16 @@ curl -X DELETE -u <user>:<password> https://localhost:4000/api/v1/api-keys/60  -
 
 #### 19.1.5 API key caching
 
-API keys are cached in memory. In typical security server configurations this does not create problems.
-However, if you have configured a setup where multiple security servers share the same `serverconf` database,
-and use multiple nodes to access REST API and execute API key management operations, the caches of different nodes
-can become out of sync.
+API keys are cached in memory, which is typically not a problem in standard security server configurations.
+However, if you have multiple security servers configured to share the same `serverconf` database
+and use multiple nodes to access the REST API and execute API key management operations, the caches of different nodes can become out of sync.
 
-For example, you may revoke an API key from node 1 but node 2 is not aware of this, and still grants access to
-REST API endpoints with this API key.
+For instance, revoking an API key from `node 1` may not be recognized by `node 2`, which can still grant access to REST API endpoints with the revoked API key. To address this issue, there are a few potential solutions:
 
-If you operate such a configuration, you need to target all REST API operations to the same security server node,
-or otherwise ensure that caching will not create problems (for example, always restart REST API modules when API key
-operations are executed).
+- **Option A:** Consider decreasing [time-to-live](ug-syspar_x-road_v6_system_parameters.md#39-management-rest-api-parameters-proxy-ui-api) value for API key cache from the default of **60 seconds** to a more lenient value. Doing so will reduce the risk of stale values being returned, thus improving security.
+- **Option B:** Direct all REST API operations to the same security server node.
```

Comment:
Let's change security server to Security Server here similar to what we do with the Central Server.